redaxo/src/core/boot.php

```diff
@@ -100,6 +98,14 @@
     rex::setProperty($key, $value);
 }
 
+// try to load cache after rex properties are available
```

Comment:
ist es wichtig dass wir nach den properties sind? gibts da eine abhängigkeit?

(oder steht hier im kommentar nur drinn dass diese zeilen code nach den code zeilen kommen, die die properties behandeln?)

Reply:
> ist es wichtig dass wir nach den properties sind? gibts da eine abhängigkeit?

ja in dem fall relevant, da vor dem laden des caches mit `rex::getConsole()` geprüft wird.